ext/nnstreamer/tensor_filter/tensor_filter_snap.cc

```diff
@@ -550,7 +550,9 @@ tensor_filter_snap::parse_custom_prop (const char *custom_prop, snap_option_s &s
       g_strstrip (option[1]);
 
       if (g_ascii_strcasecmp (option[0], "ModelFWType") == 0) {
-        if (g_ascii_strcasecmp (option[1], "TENSORFLOW") == 0) {
+        if (g_ascii_strcasecmp (option[1], "TENSORFLOWLITE") == 0) {
```

Comment:
let's fix property description also.
tensor_filter_snap::register_snap ()

Reply:
Thanks. I updated the descrition.